PoliceUK.Tests.Integrated/PoliceUkClientTests.cs

```diff
@@ -238,7 +278,7 @@ public void CrimesAtLocation_Call_With_GeoPosition_Returns_Results()
         {
             var policeApi = new PoliceUkClient();
 
-            Geoposition position = new Geoposition(51.513016, -0.10231);
+            Geoposition position = new Geoposition(51.516674, -0.176933);
```

Comment:
Changed this as the former value occasionally had no crimes for the month, which would cause the test to fail (new area is on a major travel interchange where crime is more likely).